### PR TITLE
Update Algolia geolocation docs

### PR DESCRIPTION
**Pull request by acoumb**
## Description

Added geolocation updates for Umbraco CMS - Algolia integration.

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [x] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

`Umbraco.Cms.Integrations.Search.Algolia - 2.3.0 | 3.1.0`

## Deadline (if relevant)

Anytime possible.

### CODE REVIEW

File: marketplace-and-integrations/integrations/algolia.md [hunk 1]
```diff
@@ -121,6 +121,28 @@ To customize the returned value from Umbraco CMS you would need to use a custom
 
 To extend the behavior, there are available options:
 
+### Version 2.3.0/3.1.0 and up
+
+Starting with versions `2.3.0` and `3.1.0`, Algolia comes with geolocation support for records. This comes following a community request detailed [here](https://github.com/umbraco/Umbraco.Cms.Integrations/pull/215).
```
Comment (sofietoft) [inline marketplace-and-integrations/integrations/algolia.md:126]: Let's avoid using "here" for links 💪 

```suggestion
Starting with versions `2.3.0` and `3.1.0`, Algolia comes with geolocation support for records. This comes following a [community request](https://github.com/umbraco/Umbraco.Cms.Integrations/pull/215).
```

File: marketplace-and-integrations/integrations/algolia.md [hunk 2]
```diff
@@ -121,6 +121,28 @@ To customize the returned value from Umbraco CMS you would need to use a custom
...
+
+You can read more about enabling Algolia's geolocation for records through [this](https://www.algolia.com/doc/guides/managing-results/refine-results/geolocation/#enabling-geo-search-by-adding-geolocation-data-to-records) article.
```
Comment (sofietoft) [inline marketplace-and-integrations/integrations/algolia.md:128]: Let's avoid using "this" for links 💪 

```suggestion
You can read more about enabling Algolia's geolocation for records in their [official documentation](https://www.algolia.com/doc/guides/managing-results/refine-results/geolocation/#enabling-geo-search-by-adding-geolocation-data-to-records).
```

**Review by sofietoft — CHANGES_REQUESTED** [review]
Looks great @acoumb ! 💪
I've added a couple of suggestions related to the links!

**Comment by acoumb** [comment]
Thanks @sofietoft

**Review by sofietoft — APPROVED** [review]

**Pull request merged**